Code/src/main/java/nl/utwente/group10/ui/Main.java

```diff
@@ -75,7 +75,7 @@ public void start(Stage stage) throws Exception {
         stage.setOnCloseRequest(event -> System.exit(0));
         stage.setScene(scene);
 
-        stage.setMaximized(true);
+        //stage.setMaximized(true);
```

Comment:
This file's changes seem to be just Mac-specific hotfixes. Did the Mac crew come up with a solution yet? (@Kristel-92 @TheMSB)